cli/foundation-models/system/inference/image-text-to-text/image-text-to-text-online-endpoint.sh

```diff
@@ -0,0 +1,80 @@
+set -x
+# The commands in this file map to steps in this notebook: <TODO>
+# The sample scoring file available in the same folder as the above notebook
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="llava-7"
+model_label="latest"
+
+version=$(date +%s)
+endpoint_name="image-text-to-text-$version"
+
+# Todo: fetch deployment_sku from the min_inference_sku tag of the model
+deployment_sku="Standard_NC6s_v3"
+
+# Prepare data for deployment
+data_path="./data_online"
+python ./prepare_data.py --data_path $data_path --mode "online"
```

Comment:
If we are only supporting online inference then we do not need the --mode argument. It doesn't seem to be parsed in prepare_data.py

Reply:
Yes, "--mode" is not used, thank you for the catch.